contents/blog/10x-engineers-do-user-interviews.md

```diff
@@ -0,0 +1,134 @@
+---
+date: 2023-02-27
+title: "10x engineers talk to users"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - luke-harries
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product engineers
+---
+
+Most people think becoming a 10x engineer means improving their typing speed, buying more monitors, or using the latest framework where everything is a macro. These hacks might improve efficiency somewhat, but you'll get a 1.5x boost at most.
+
+So, where does the next 8.5x come from? By zooming out and solving for user impact. You, as an engineer, need to deeply understand (and care) about your users. In doing so, you'll generate true leverage by choosing the right problem, building the right solution, and solving their "hair-on-fire" problems. 
+
+The best way to do this? Speak directly to users.
+
+## Solving the information bottleneck
+
+In traditional tech companies, information flows from the user to the product manager (PM) by an interview, and then from the PM to the engineer through a bunch of tickets:
+
+![PM to engineer](../images/blog/10x-engineers-do-user-interviews/pm-to-engineer.png)
+
+At each stage, important context is lost. Information flows one way, so you can’t go deeper on the important areas, and engineers lose agency in what they're building and why. In such an environment, speed is the only logical optimization for an engineer.
+
+Here's what it looks like when engineers talk directly to users:
+
+![Direct user-Engineer Interaction](../images/blog/10x-engineers-do-user-interviews/direct-interaction.png)
+
+Information moves quickly and in both directions, boosting the signal. By increasing the signal you can quickly cut scope, realize you’re tackling the wrong problem, or come up with a more elegant solution – something engineers can do most effectively because they have the background on the existing architecture and components. 
+
+Companies still need product managers (I'm the Head of Product at PostHog after all 😅), but we ship faster and better because our [product engineers](/blog/what-is-a-product-engineer) speak directly to users. And, because our product managers aren't spending all their time on tickets, they have time to surface new opportunities or build out [our bigger vision](/blog/helping-engineers-to-product).
+
+> 💬 **Sidenote:** PostHog's session recording feature came about because Karl, one our engineers, took the time to talk to a user and realized the best solution was integrating recordings with our existing product analytics tool. One hackathon later, we had a prototype. Session recording is now a core part of our product and was a key driver in our strategy shift from a pure, self-hosted product analytics tool to an all-in-one product OS.
```

Comment:
```suggestion
> 💬 **Sidenote:** PostHog's session recording feature came about because Karl, one of our engineers, took the time to talk to a user and realized the best solution was integrating recordings with our existing product analytics tool. One hackathon later, we had a prototype. Session recording is now a core part of our product and was a key driver in our strategy shift from a pure, self-hosted product analytics tool to an all-in-one product OS.
```